.travis.yml

```diff
@@ -14,17 +14,23 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-language: minimal
+os: linux
+dist: focal
+language: shell
 
 git:
   submodules: false
   depth: false
 
 install:
- - git describe
- - make env
+  - git describe
+  - python3 -m pip install flake8
```

Comment:
These tools should be installed as part of the `make env` environment.

Reply:
Do we want our runtime to be larger than it needs to be?  These are not runtime requirements.  End users do not need these tools.  These are dev/test requirements only.

Reply:
We could add a `make test-env` or `make dev-env`?